### PR TITLE
[cli] add documentation for the DNS CLI commands

### PR DESCRIPTION
**Pull request by kylorene**

### CODE REVIEW

File: third_party/mbedtls/repo/include/mbedtls/ssl.h [hunk 1]
```diff
@@ -1514,7 +1514,7 @@ MBEDTLS_DEPRECATED extern int (*mbedtls_ssl_hw_record_finish)(
  *
  * \return              a string containing the ciphersuite name
  */
-const char *mbedtls_ssl_get_ciphersuite_name( const int ciphersuite_id );
+const char *mbedtls_ssl_get_ciphersuite_name( int ciphersuite_id );
```
Comment (abtink) [inline third_party/mbedtls/repo/include/mbedtls/ssl.h:1517]: Remove these? 
Generally no changes in `third_party` sources.

**Review by abtink — COMMENTED** [review]
LGTM. Thanks. 
One suggestion below:

**Comment by Vyrastas** [comment]
I'm working on getting @kylorene covered by the CLA.

**Review by Vyrastas — APPROVED** [review]

**Review by jwhui — APPROVED** [review]
Thanks! 👍🏼

**Pull request merged**